hscontrol/types/config_test.go

```diff
@@ -333,3 +334,59 @@ tls_letsencrypt_challenge_type: TLS-ALPN-01
 	err = LoadConfig(tmpDir, false)
 	assert.NoError(t, err)
 }
+
+// OK
+// server_url: headscale.com, base: clients.headscale.com
+// server_url: headscale.com, base: headscale.net
```

Comment:
It is said elsewhere that server url must not be a suffix of the base_domain. To me, server is a suffix of base here -- yes, additionally they are equal. Perhaps the wording using "suffix" needs to be adjusted?

Reply:
you are write, it should probably be reformulated since we allow different domains, open to proposals.

Reply:
See updated error message. I think the new one is more intuitive, see if you like it better